doc/devel/release_guide.rst

```diff
@@ -382,3 +382,14 @@ In addition, announcements should be made on social networks (twitter
 via the ``@matplotlib`` account, any other via personal accounts).
 `NumFOCUS <https://numfocus.org/>`__ should be contacted for
 inclusion in their newsletter.
+
+
+Conda packages
+==============
+
+The conda packaging scripts for Matplotlib are hosted at
```

Comment:
Even for the version distributed on the main conda channel?
I think a sentence about why this info could be useful (so it doesn't confuse folks who don't need to know) could be helpful here.

Reply:
Reworded.

Note also, that this is part of the release guide, which states at its top

![grafik](https://user-images.githubusercontent.com/2836374/111499787-96f30680-8743-11eb-84f0-b95faf520749.png)